doc/modules/mod_event_pusher.md

```diff
@@ -0,0 +1,36 @@
+### Module Description
+This module is a generic interface for event pushing backends.
+It defines a single hook, `push_event/3` that forwards the event to all registered backends.
+Each backend decides how and if to handle the event in its `push_event/2` implementation.
+
+The events are standardized as records that can be found in `mod_event_pusher_events.hrl` file.
+Common events like user presence change (offline and online), chat and groupchat messages (incoming and outgoing) are already hooked up to the frontend via `mod_event_pusher_hook_translator` that translates various hooks into specific calls of the `push_event/3` hook.
```

Comment:
Veto

Reply:
Veto of veto. Translator does not translate anything into **calls of `push_event/3` hook**. If we want to be consistent here, translator should execute `push_event` hook.

Reply:
The translator is a strongly coupled part of the `mod_event_pusher` so I don't see anything necessarily wrong with calling the hook (because push_event *is* a hook) directly here.

Reply:
True, but in such case `push_event/3` does not integrate with the translator as a hook but as a function. This doc change implies that translator integrates via hooks mechanism.

Reply:
From outside perspective that's what translator does, i.e. because push_event is called in a hook implementation (e.g. of `filter_packet`), it's just as if `push_event` was called via `ejabberd_hooks:run`

Reply:
For an average user indeed it doesn't make a difference but from developer perspective it is a too strong simplification. If you'd like to keep this semantics, I think the more precise statement would be something like "`mod_event_pusher_hook_translator` which is a proxy between various hooks and `push_event/3` hook handler."